### PR TITLE
Fix homepage and deposit links

### PR DESCRIPTION
**Pull request by lfarrell**
Departments link still isn't set.

### CODE REVIEW

File: app/views/hyrax/homepage/_unc_masthead.html.erb [hunk 1]
```diff
@@ -8,9 +8,12 @@
   <div class="link-tabs col-xs-12 col-sm-12">
     <ul class="list-unstyled pull-right">
       <li><button><%= link_to t('hyrax.links.home'), "/" %></button></li>
-      <li><button><%= link_to t('hyrax.links.browse_collections'), "/" %></button></li>
+      <li><button><%= link_to t('hyrax.links.browse_collections'), main_app.search_catalog_path %></button></li>
```
Comment (bbpennel) [inline app/views/hyrax/homepage/_unc_masthead.html.erb]: None of the buttons do anything when I click on them in firefox, but they work in chrome

File: app/assets/stylesheets/unc_custom.css.scss [hunk 2]
```diff
@@ -264,4 +264,9 @@ body {
       margin-top: 5px;
     }
   }
+}
+
+// Make sure date picker is the top element
+.ui-datepicker-div {
```
Comment (bbpennel) [inline app/assets/stylesheets/unc_custom.css.scss]: Is this a bonus fix to make the datepicker not go under form elements?

Comment (lfarrell) [reply app/assets/stylesheets/unc_custom.css.scss]: yes

File: app/assets/javascripts/unc_custom.js [hunk 3]
```diff
@@ -1,6 +1,22 @@
...
         dateFormat: 'yy-mm-dd'
     });
+
+
+    // Only show student paper options in modal when clicking "Student Papers" link on homepage
+    var all_work_types = $('form.new-work-select .select-worktype');
+
+    $('#student-papers-work-types').on('click', function (d) {
+        all_work_types.filter(function(index) {
+            return index === 1 || index === 2 || index >= 4;
```
[the remark this answers is not in the document]
Comment (lfarrell) [reply app/assets/javascripts/unc_custom.js]: There's a method in one of the controllers/presenters, which we're going to have to override anyway to display the affiliation beside the creator. I'll see how it goes

File: app/views/hyrax/homepage/_unc_masthead.html.erb (hunk 1 above)
Comment (lfarrell) [reply app/views/hyrax/homepage/_unc_masthead.html.erb]: Hmm, apparently firefox doesn't let you wrap a link in a button.

Comment (bbpennel) [reply app/views/hyrax/homepage/_unc_masthead.html.erb]: These are working for me now

File: app/assets/javascripts/unc_custom.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (lfarrell) [reply app/assets/javascripts/unc_custom.js]: Should be less fragile now.

**Pull request merged**